examples/BME_280_BRZO_I2C_Test/BME_280_BRZO_I2C_Test.ino

```diff
@@ -24,6 +15,27 @@ Gnd (Ground)        ->  Gnd
 SDA (Serial Data)   ->  D2 on ESP8266
 SCK (Serial Clock)  ->  D1 on ESP8266
 
+
+Copyright (c) 2015-2021 Tyler Glenn, Alex Shavlovsky, et al.
```

Comment:
To be honest I think I would just like to have the date the original file was written. I don’t really see a need for it to be updated. The copyright really only shows that I wrote this code first, so the initial date is really the only one that matters. The github history can show the rest…. Unless someone can provide a valid argument against it….

Reply:
Hey @finitespace,

Sorry for the delay.

I have changed the copyright dates for the merge request according to your comments.

Wherever there was a copyright stated in the files I used that even if the created date in the comment statet the previous year.  
However, in one case where the copyright statement claimed 2016 but the file was create 2017 (as stated in the comment inside the file), I took the more recent year.

I am not a lawywer and definitely not a copyright expert. I presume that the stated year refers to the time of release, not the time the file was initially created.
That would explain the cases where, e.g., files were created 2015 but the copyright statement states 2016.

Cheers, @tinxx

Reply:
Hey @finitespace, any updates?